Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -245,8 +245,8 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         removeCompletion()
 
         let isAtEnd = selectedTextRange?.start == endOfDocument
-        let isEmpty = lastReplacement?.isEmpty ?? true
-        if !isEmpty, isAtEnd, markedTextRange == nil {
+        let isReplacing = lastReplacement != nil
+        if isReplacing, isAtEnd, markedTextRange == nil {
```

Comment:
Is it more Swifty to not check for `lastReplacement != nil` like:
`if let _ = lastReplacement, isAtEnd, markedTextRange == nil {`